commands/create-account.js

```diff
@@ -32,6 +38,26 @@ module.exports = {
 
 async function createAccount(options) {
     // NOTE: initialBalance is passed as part of config here, parsed in middleware/initial-balance
+    // periods are disallowed in top-level accounts and can only be used for subdomains
+    const splitAccount = options.accountId.split('.');
+    if (splitAccount.length === 2) {
+        // TLA (bob-at-least-maximum-chars.test)
+        if (splitAccount[0].length < TLA_MIN_LENGTH) {
+            console.log(`Top-level accounts (not ending in .near, .test, etc) must be greater than ${TLA_MIN_LENGTH} characters`);
+            return;
+        }
+    } else if (splitAccount.length === 3) {
```

Comment:
length isn't limited to 3, can be easily `foo.bar.baz.alice.near` if you have `bar.baz.alice.near` as `masterAccount`

Reply:
thanks, this is helpful feedback